app/models/practice.rb

```diff
@@ -1,10 +1,12 @@
 class Practice < ApplicationRecord
   acts_as_list
   # has_attached_file :main_display_image, styles: { medium: "300x300>", thumb: "100x100>" }, default_url: "/images/:style/missing.png"
-  has_attached_file :main_display_image, default_url: "/images/:style/missing.png"
-  has_attached_file :origin_picture, default_url: "/images/:style/missing.png"
+  has_attached_file :main_display_image
+  has_attached_file :origin_picture
   validates_attachment_content_type :main_display_image, content_type: /\Aimage\/.*\z/
 
+  belongs_to :user, optional: true
```

Comment:
Are we worried about this being a `has_and_belongs_to_many`? Just wondering if multiple contributors will be a thing.

Reply:
we will have to ask andy that soon. for now, it belongs to one user